src/components/MainTabs/MainTabs.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Home from "pages/Home/Home";
+import Offers from "pages/Offers/Offers";
+import { Route, Switch } from "react-router";
+
+const MainTabs = () => (
```

Comment:
The assumption of this component is that it would never handle routes other than what was defined in the Route component that wrapped it.
```jsx
<PrivateRoute exact path={["/home", "/offers"]}>
  <MainTabs />
</PrivateRoute>
```
Does this assumption need to be made more explicit? E.g. throwing an error if an illegal route comes through?